requirements/base.txt

```diff
@@ -4,18 +4,116 @@
 #
 #    pip-compile --output-file=requirements/base.txt requirements/base.in
 #
-
 asgiref==3.8.1
     # via django
 backports-zoneinfo==0.2.1 ; python_version < "3.9"
     # via
     #   -c requirements/constraints.txt
     #   django
-django==4.2.16
+certifi==2025.1.31
+    # via requests
+cffi==1.17.1
+    # via
+    #   cryptography
+    #   pynacl
+charset-normalizer==3.4.1
```

Comment:
Our edx-platform uses charset-normalizer==2.0.12. I can continue with the others. Please take in account that its important to maintain the same versions because it can create several conflict in our enviroments.